media/js/firefox/australis/growth-firstrun-test2.js

```diff
@@ -247,7 +232,6 @@ function onYouTubeIframeAPIReady() {
 
         // only track if playing or ended
         if (state) {
-            window.gaTrack(['_trackEvent', '/firstrun/ Optimization f36', videoName, state]);
```

Comment:
Can remove this conditional and the `videoName` declaration above.

Additionally, how is this video now being tracked?